pages/learn/deploy/release-strategy/update-strategies.md

```diff
@@ -8,19 +8,22 @@ excerpt: Choosing an update strategy for your {{ $names.company.lower }} devices
 With the {{ $names.company.lower }} device supervisor version 1.3, we added the ability to choose the update strategy on devices, that is, the order and way in which the steps to perform an update are executed. You can check whether your Supervisor has the appropriate version in the "Supervisor version" entry in the device dashboard page.
 These update strategies allow users to choose between four modes that are suited for different applications, depending on available resources and the possible need to have a container running at all times.
 
-Update strategies can be applied in the [Configuration][configuration] section. The two variables that are involved are:
+Update strategies can be applied by setting a [docker-compose label](labels). Two labels that are involed are:
```

Comment:
```suggestion
Update strategies can be applied by setting a [docker-compose label](labels). Two labels that can be invoked are:
```

Reply:
I think I meant "involved" here :p, nice catch @klutchell. Do you think "invoked" works better? Can you invoke a label?

Reply:
I think "involved" makes more sense in the full context of the phrase. "Update strategies can be applied by setting docker-compose label. The two labels that are involved are"